doc/examples/how_to_dmms/how_to_dmms.ipynb

```diff
@@ -0,0 +1,566 @@
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>parameterScale</th>\n",
+       "      <th>lowerBound</th>\n",
+       "      <th>upperBound</th>\n",
+       "      <th>nominalValue</th>\n",
+       "      <th>estimate</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>parameterId</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>alpha</th>\n",
+       "      <td>lin</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>15.0</td>\n",
+       "      <td>1.3</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>delta</th>\n",
+       "      <td>lin</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>15.0</td>\n",
+       "      <td>1.8</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>beta</th>\n",
+       "      <td>lin</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>15.0</td>\n",
+       "      <td>0.9</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>net3_ps</th>\n",
+       "      <td>lin</td>\n",
+       "      <td>-inf</td>\n",
+       "      <td>inf</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            parameterScale  lowerBound  upperBound  nominalValue  estimate\n",
+       "parameterId                                                               \n",
+       "alpha                  lin         0.0        15.0           1.3         1\n",
+       "delta                  lin         0.0        15.0           1.8         1\n",
+       "beta                   lin         0.0        15.0           0.9         1\n",
+       "net3_ps                lin        -inf         inf           NaN         1"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "petab_problem.parameter_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb0d08d2-088b-4850-96e2-d0c12e1605eb",
+   "metadata": {},
+   "source": [
+    "### Array input data\n",
+    "\n",
+    "The PEtab problem specified under ``extensions_config`` that there were two files of array data to be added to the problem:\n",
```

Comment:
`extensions_config` is not a part of the PEtab spec, but rather just how one would access the extension config when using the library. I guess this might be clearer
```suggestion
    "The PEtab problem YAML file, in the `petab_sciml` section of the `extensions` field, specifies two files of array data to be included in the problem:\n",
```

and I would rather paste the "real" YAML extensions section here, not the JSON, since the YAML is what most users will work with.